drivers/util.py

```diff
@@ -637,6 +637,17 @@ def zeroOut(path, fromByte, bytes):
     return True
 
 
+def wipefs(blockdev):
+    "Wipe filesystem signatures from `blockdev`"
+    cmdlist = ["wipefs", "-a", blockdev]
```

Comment:
When we invoked dd it was done via `util.CMD_DD` which gave `/bin/dd` as an explicit path.

I think it would be good to follow the same procedure and create `util.CMD_WIPEFS` with an explicit path (I believe that would be `/usr/sbin/wipefs`) just to maintain the same level of rigour.

Other than that I don't see an issue.

Reply:
I'm curious about any upsides with this approach, other than consistency with existing code.  As for downsides I see:
* hardcoding paths decreases portability
* using a variable defined somewhere else makes the code less readable

The init system is responsible to set a proper `PATH` in environment (and nowadays `systemd` ensures this even when launching the service from the CLI).  If we don't trust the environment we're just doomed anyway: anything could be achieved with a `LD_PRELOAD`, including replacing the path passed to `execve`.

Reply:
Portability isn't really an issue for this code as it's part of an appliance use-case.

You're right that _at present_ someone could use LD_PRELOAD to do anything they could do by subverting the PATH, but that may not always be the case. Should future hardening work move this call into a situation of elevated privilege, the dynamic linker would ignore any LD_PRELOAD with a path separator, rendering it useless without additional PATH subversion (which a hard-coded path would prevent).

I'm willing to let this go at present, but it's something we'll need to consider if we decide to harden this path in the future.